CHANGES.rst

```diff
@@ -84,6 +84,8 @@ astropy.modeling
   ``inputs`` and ``outputs`` are generated. ``Model.inputs`` and ``Model.outputs``
   are now settable which allows renaming them on per user case. [#9298]
 
+- Add ``BlackBody`` model.  [#9282]
```

Comment:
p.s. The usual convention is to show PR number at the end of the last line of change log entry. See other entries as an example.